src/ArrayHelper.php

```diff
@@ -1330,6 +1330,29 @@ public static function getObjectVars(object $object): array
         return get_object_vars($object);
     }
 
+    /**
+     * Replace key in array.
+     *
+     * @param array $array Source array.
+     * @param int|string $from Key to replace.
+     * @param int|string $to Key to replace with.
+     *
+     * @return array The result array.
+     */
+    public static function replaceKey(array $array, int|string $from, int|string $to): array
+    {
+        if (!isset($array[$from])) {
+            return $array;
+        }
+
+        $result = [];
+        foreach ($array as $key => $value) {
```

Comment:
I don't think `foreach` is needed here. Also, it doesn't remove old key.

```php
if (array_key_exists($from, $result)) {
   $result[$to] = $value;
   unset($result['from'])
}
```

Reply:
This approach change order of items.

Reply:
Right. The code makes sense then.